src/test/regress/sql/global_cancel.sql

```diff
@@ -1,3 +1,16 @@
+-- add coordinator in idempotent way, this is needed so we get a nodeid
+SET client_min_messages TO ERROR;
+SELECT 1 FROM master_add_node('localhost', :master_port, groupid => 0);
+RESET client_min_messages;
+
+-- Kill maintenance daemon so it gets restarted and gets a gpid containing our
+-- nodeid
+SELECT pg_terminate_backend(pid)
```

Comment:
probably citus_add_node should do this automatically at some point. But given adding the coordinator is not a very common thing, it is not becoming an issue